services/common/src/redux/slices/spatialDataSlice.ts

```diff
@@ -6,52 +6,90 @@ import { formatDate } from "../utils/helpers";
 import { IGeoJsonFeature } from "@mds/common/interfaces/document/geojsonFeature.interface";
 import { ISpatialBundle } from "@mds/common/interfaces/document/spatialBundle.interface";
 
+interface ItempMineDocumentWithGeomarkId extends IMineDocument {
```

Comment:
I am trying to work out what an Itemp is? Typo?

Reply:
it's a temporary MineDocument with a geomarkId